charts/influxdb2/README.md

```diff
@@ -1,56 +1,49 @@
 # InfluxDB
```

Comment:
Do all sections in the InfluxDB readme also apply to InfluxDB Enterprise? It's not clear at the beginning that the readme is for both until you get down to the Enterprise section. Thoughts?

Reply:
It's for both, definitely. Enterprise has a few extra options.

I'd prefer to split these up, to be honest

Reply:
Makes sense to split. For now, I added a note at top to refer to "InfluxDB Enterprise Helm chart"